proto/rill/local/v1/api.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package rill.local.v1;
+
+import "google/protobuf/timestamp.proto";
+
+service LocalService {
+  // Ping returns the current time.
+  rpc Ping(PingRequest) returns (PingResponse) {}
+
+  // GetMetadata returns information about the local Rill instance.
+  rpc GetMetadata(GetMetadataRequest) returns (GetMetadataResponse) {}
+
+  // GetVersion returns details about the current and latest available Rill versions.
+  rpc GetVersion(GetVersionRequest) returns (GetVersionResponse) {}
+}
+
+message PingRequest {}
+
+message PingResponse {
+  google.protobuf.Timestamp time = 1;
+}
+
+message GetMetadataRequest {}
+
+message GetMetadataResponse {
+  string instance_id = 1;
+  string project_path = 4;
```

Comment:
just curious about the order number here ?

Reply:
Fixed :)